app/src/main/java/com/github/factotum_sdp/factotum/ui/roadbook/RoadBookFragment.kt

```diff
@@ -74,9 +89,50 @@ class RoadBookFragment : Fragment(), MenuProvider {
     }
     override fun onCreateMenu(menu: Menu, menuInflater: MenuInflater) {
         menuInflater.inflate(R.menu.main, menu)
```

Comment:
Function is a bit long, could used a bit of refactoring.

Reply:
I agree.